Mathlib/RingTheory/Localization/Module.lean

```diff
@@ -70,83 +148,66 @@ variable [hA : IsLocalization (Algebra.algebraMapSubmonoid A S) Aₛ]
 
 open Submodule
 
+theorem isLocalizedModule_of_isScalarTower :
+    IsLocalizedModule S (IsScalarTower.toAlgHom R A Aₛ).toLinearMap where
```

Comment:
We should upgrade this to an iff and this should live in `Mathlib/Algebra/Module/LocalizedModule.lean`.

Reply:
I proved
```
lemma isLocalizedModule_iff_isLocalization (R') [CommRing R'] [Algebra R R'] :
    IsLocalizedModule S (Algebra.ofId R R').toLinearMap ↔ IsLocalization S R' := sorry
```
in [this branch](https://github.com/leanprover-community/mathlib4/compare/xfr-basis_islocalizedmodule...jyxu/isLocalizedModule?expand=1), which seems also missing. Feel free to merge!

Reply:
Merged. Thanks!

Reply:
Could you add `IsLocalization (Algebra.algebraMapSubmonoid A S) Aₛ ↔ IsLocalizedModule S (IsScalarTower.toAlgHom R A Aₛ).toLinearMap` which I think is @erdOne's suggestion? It's analogous to `isLocalizedModule_iff_isLocalization` and shouldn't be much more difficult to prove ... And we could even derive `isLocalizedModule_iff_isLocalization` from it, I think.